generator/source/faq/settings.rst

```diff
@@ -52,8 +52,9 @@ sawtooth.consensus.max_wait_time
 sawtooth.consensus.min_wait_time
     Minimum devmode consensus wait time, in seconds
 
-sawtooth.consensus.pbft.peers
-    JSON list of each peer node's public key. Only required PBFT setting.
+sawtooth.consensus.pbft.members
+    JSON list of public keys belonging to each peer node that is a member of the
```

Comment:
I would change `peer node` to something else, to be consistent with your other changes. Member nodes?

Reply:
I would just remove the word "peer", because the rest of the sentence has the necessary info on being a member of the network.

Reply:
Fixed